#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros_utils'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("grid_size", double_t, 0, "Sampling grid size", 0.01, 0.001, 1.0)
exit (gen.generate (PACKAGE, "jsk_pcl_ros_utils", "PolygonPointsSampler"))
